codegen/debugger.go

```diff
@@ -320,6 +320,10 @@ func (d *debugger) yield(ctx context.Context, scope *ast.Scope, node ast.Node, v
 
 		err = req.Solve(ctx, d.cln, MultiWriter(ctx))
 		if err != nil {
+			// If debugger has an error, continue to exit.
+			if d.err != nil {
```

Comment:
I used `if d.err != nil` to be consistent with the check [above](https://github.com/openllb/hlb/blob/master/codegen/debugger.go#L311). I'm also okay with checking for `ErrDebugExit` explicitly here (`errors.Is(err, ErrDebugExit)`).

Reply:
it seems like if we have any debugger error, we should be exiting, so it probably does not make sense to special case for ErrDebugExit?

Reply:
I think the only value for `d.err` currently is `ErrDebugExit`, but I agree it's better to check for any debugger error here, if new errors get added later.